src/mixins/date-mixin.ts

```diff
@@ -173,6 +173,18 @@ export default class DateMixin extends Mixins(CommonMixin) {
     return `${dateStr} at ${timeStr} Pacific time`
   }
 
+  /**
+   * Converts a Date object to an API datetime string.
+   * @example 2021-08-05T16:56:50Z -> 2021-08-05T16:56:50+00:00
+   */
+  dateToApi (date: Date): string {
+    // safety check
+    if (!isDate(date) || isNaN(date.getTime())) return null
+
+    // replace "Zulu" timezone abbreviation with UTC offset
+    return date.toISOString().replace('Z', '+00:00')
+  }
+
```

Comment:
Moved here to be in same order as this same mixin in Filings UI (with the eventual goal of having a shared mixin or service that could be imported everywhere).